tests/layer_tests/mo_python_api_tests/test_mo_convert_complex_params.py

```diff
@@ -150,7 +171,14 @@ def create_tf_param_res_model(self, tmp_dir):
                        'Input2': LayoutMap(source_layout="nc??", target_layout=Layout("n??c")),
                        'Input3': LayoutMap(source_layout="abcd", target_layout="acdb")}},
             'params_ref': {'layout': "Input1(nchw->nhwc),Input2(nc??->n??c),Input3(abcd->acdb)"}},
-
+        {'params_test': {'input': [PartialShape([2, 3, 4]), [2, 3, 4], [Dimension(2), Dimension(3), Dimension(4)]]},
+         'params_ref': {'input_shape': "[2,3,4],[2,3,4],[2,3,4]", 'input': 'Input1,Input2,Input3'}},
+        {'params_test': {'input': [np.int32, Type(np.int32), np.int32]},
+         'params_ref': {'input': 'Input1{i32},Input2{i32},Input3{i32}'}},
+        {'params_test': {'input': [InputCutInfo(None, [1], np.int32, [10]),
```

Comment:
could you please consider suggestion with deffault values?
https://docs.python.org/3/library/collections.html#collections.namedtuple

InputCutInfo = namedtuple("InputInfo", ["name", "shape", "type", "value"], defaults=[None, None, None, None])

after that this case can be written like:
InputCutInfo(shape=[1], type=np.int32, value=[10])